client/components/recycleAndReuseComponents/Steps/StepThree/Geolocation/utils.js

```diff
@@ -97,44 +116,58 @@ export const getNearestNonBlueBinFacilities = (items, { lat, lng }) => {
 	const results = [];
 
 	for (const item of items) {
-		const validLocations = [];
+		let distance = null;
+		let nearestLocation = null;
 
 		for (const place of physicalChannels) {
 			if (
 				place.categories_accepted.includes(item.category) &&
 				place.type.includes(item.condition)
 			) {
-				validLocations.push({
-					id: place.id,
-					postal: place.postcode,
-					distance:
-						Math.round(
-							calcCrow(
-								lat,
-								lng,
-								place.latitude,
-								place.longitude,
-							) * 100,
-						) / 100,
-					latitude: place.latitude,
-					longitude: place.longitude,
-					address: place.address,
-					channel_name: place.channel_name,
-					operating_hours: place.operating_hours,
-					contact: place.contact,
-					website: place.website,
-					categories_accepted: place.categories_accepted,
-					organisation_name: place.organisation_name,
-					type: place.type,
-				});
+				let calcDist = Math.round(
```

Comment:
is this change from 127 - 150 necessary?

to my understanding, the changes are
1. handling for no valid facilities
2. making the `items` for that facility an array

if so, can probably just updated old lines 133-136

Reply:
Sure, will revert to previously used logic to determine nearest location. The changes will mainly target point 2. Point 1 is covered in a separate issue.